csunplugged/topics/models.py

```diff
@@ -11,7 +11,7 @@ class GlossaryTerm(models.Model):
 
     #  Auto-incrementing 'id' field is automatically set by Django
     slug = models.SlugField(unique=True)
-    term = models.CharField(max_length=200, unique=True)
+    term = models.CharField(max_length=200)
```

Comment:
Worked perfectly! Doesn't allow duplicates (two different files with the same title) but they can be created as `null` to begin with. Ready for re-review.